include/circt/Dialect/FIRRTL/FIRRTLOps.h

```diff
@@ -55,6 +55,9 @@ bool isConstant(Value value);
 /// constant values
 bool isConst(Type type);
 
+/// Returns true if the type is or contains a 'const' type.
+bool containsConst(Type type);
+
```

Comment:
Not sure if you're using this in future PRs, but if this only has one use then it might as well be static in that cpp file.  It helps link times and library code bloat.  Besides that, it seems like it should go in `FIRRTLTypes.h` or `FIRRTLUtils.h`.